README.md

```diff
@@ -4,6 +4,8 @@ Spark
 Spark is a public domain FHIR server developed in C#, initially built by Firely and as of recently being
 maintained by Kufu.
 
+**DISCLAIMER: This is meant as an reference implementation, and should never be used as is in a production environment.**
```

Comment:
Could you change it to:

The web projects spark and spark-classic are meant as reference implementations and should never be used out of the box in a production environment without adding as a minimum security features.

Reply:
Agree. Commit updated and force-pushed (!).